src/classes/RD_RecurringDonations.cls

```diff
@@ -1214,4 +1543,92 @@ public class RD_RecurringDonations {
         }
         return calcDate;
     }
+
+    /*******************************************************************************************************
+    * @description Returns true if the RD changes should trigger recalculation of the child opportunities
+    * @param new The updated Recurring Donation
+    * @param old The original Recurring Donation before updates
+    * @return boolean
+    */
+    @testVisible
+    private static Boolean needsRecalculation(npe03__Recurring_Donation__c newRD, npe03__Recurring_Donation__c oldRD) {
+
+        if(newRD.npe03__Organization__c != oldRD.npe03__Organization__c
```

Comment:
not sure if this is because you copied existing code, but I our standard is space between if (...)